### PR TITLE
[Docs] Fix documentation for the `roles` field in the Moderated Sessions join policy reference

### PR DESCRIPTION
**Pull request by rudream**
## Purpose

The current explanation for the `roles` field under the `join_sessions` policy in the docs is incorrect. It states that `roles` field is a list of roles who can join sessions under the policy. In actuality, it is a list of roles whose sessions can be joined with the policy.

### CODE REVIEW

File: docs/pages/access-controls/guides/moderated-sessions.mdx [hunk 1]
```diff
@@ -101,7 +101,7 @@ The following are required options for `join_sessions`:
 |Option|Type|Description|
 |---|---|---|
 |`name`|String|The name of the allow policy|
-|`roles`|[]String|A list of names for Teleport roles that this policy applies to. Users with this role are eligible to join a Moderated Session under this policy.|
+|`roles`|[]String|A list of names of Teleport roles whose sessions this policy applies to. Moderated Sessions created by users with these roles can be joined under this policy.|
```
Comment (zmb3) [inline docs/pages/access-controls/guides/moderated-sessions.mdx:104]: Is this true for all sessions or only moderated ones?

Comment (rudream) [reply docs/pages/access-controls/guides/moderated-sessions.mdx:104]: Just tested and confirmed that it's for all sessions.

**Review by zmb3 — APPROVED** [review]

**Review by ptgott — APPROVED** [review]

**Comment by ptgott** [comment]
@rudream I've added all possible backport labels to this PR, but please feel free to correct these.

**Comment by zmb3** [comment]
This is good for backport all the way to v10 👍

**Pull request merged**